registry/build/Dockerfile

```diff
@@ -5,10 +5,11 @@ SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]
 ENV DOCKER_BUILDTAGS include_oss include_gcs
 ENV DISTRIBUTION_VERSION v2.5.1
 
-RUN go get github.com/tools/godep ; \
-    mkdir src\github.com\docker ; \
+RUN mkdir src\github.com\docker ; \
     cd src\github.com\docker ; \
     git clone https://github.com/docker/distribution ; \
     cd distribution ; \
     git checkout $env:DISTRIBUTION_VERSION ; \
+    try { go get github.com/tools/godep } catch { } ; \
+    go get github.com/tools/godep ; \
```

Comment:
Maybe some DNS issues inside a container. But if it works for you I'm fine with this workaround until we find the root cause of if containers or golang behave better in the future.